frontend/src/components/tracks/index-ixd.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+
+import Header from './header/index';
+import Footer from '../shared/footer/index';
+import TrackImage from './track-image/index';
+import WhatYouLearn from './what-you-learn/index';
+import OfferResponsibilities from './offer-responsibilities/index';
+import Expectations from './expectations/index';
+import { or, header, expectations } from './data/pm';
```

Comment:
You should import the IxD data file here. ;) 

```suggestion
import { or, header, expectations } from './data/ixd';
```